src/languages/javascript.js

```diff
@@ -360,7 +360,8 @@ export default function(hljs) {
       /\b/,
       noneOf([
         ...ECMAScript.BUILT_IN_GLOBALS,
-        "super"
+        "super",
+        "import"
```

Comment:
When the functional version of import is used `import()` should we not highlight it as a function?  Thoughts?